PublicationsRouter/v2/sword-out/REPOSITORY.md

```diff
@@ -18,6 +26,18 @@ Router is only certified to work with EPrints 3.3.
 EPrints is configured by default to accept incoming requests via SWORDv2, so the router's deposit mechanism
 will automatically work.  
 
+### Eprints RIOXX Plugin
+
+A brief description of the plugin is available here: [RIOXX PubRouter](http://wiki.eprints.org/w/Jisc_Publications_Router)  
+In summary, the following would need to be done:
```

Comment:
You need to make clear that this needs to be done to ENABLE use of Eprints RIOXX.